usecases/auth/authorization/rbac/manager.go

```diff
@@ -75,7 +87,7 @@ func (m *manager) GetRoles(names ...string) ([]*models.Role, error) {
 				return nil, err
 			}
 			if len(polices) == 0 {
-				return nil, fmt.Errorf("%w: %s", authorization.ErrRoleNotFound, name)
+				continue
```

Comment:
**thought:** here rises the question, if you ask about list of roles and one of them is not found, shall error be returned or ignore 🤔 , i'd say we return error and reject the whole request but wdyt from UX prospective ?

Reply:
Agreed! At the API layer, the handler interprets a `len(roles) == 0` result inside the `getRole` method as an error and so returns a 404 not found error there. Here, we're at the `RbacManager` level where the return is `[]*models.Role` so from the PoV of the caller of this method, returning a partial result in such a case is preferable